### ISSUE TITLE
Version Metadata for datasets

### ISSUE THREAD

**Issue by askarlupka** [EDRN/labcas-ui#152]
## 🤔 Tell Us About the Feature

It is common to disclose which version of the data was used for analysis. Therefore, if the data was changed after the first download and someone else downloads the second version and is unable to replicate it, it may be because the version is different. Having a version metadata would be useful for being compliant with data re-use expectations. Most of the datasets would be version 1, but it would be useful to know if they have ever been updated/adjusted. 

## 🎇 What Solution You'd Like

Add one more row that includes the version number:
![image](https://user-images.githubusercontent.com/54122306/192007242-6abae4a7-6c75-47d2-9b7a-a69a3129bb8f.png)

## 〽️ Alternative Ideas

If the ID is updated for each version, then some clarification on that would be useful. Or adding _v1, at then end of the id would help make it clearer.

**Comment by yuliujpl** [comment]
@askarlupka  good suggestion! We do have the version number in the backend, now we just need to display it. @hoodriverheather  @nutjob4life  would you agree that we should display version number for all of our datasets/files?

**Comment by askarlupka** [comment]
@yuliujpl I don't know if this warrants it's own issues, but in addition the date of when the dataset was published online would be very useful. At least one of the datasets is not associated with a publication so it's unknown when the data was generated or how long it has been available.

**Comment by askarlupka** [comment]
I've found the date information for the dataset I was looking at. The date is available within each individual file's metadata. Looking back at it a second time, I also see the dataset's version in addition to the file's version. Since the dataset version is one level higher than the file's version, I think that would make better sense to include the dataset metadata on it's landing page instead of within a file's characteristics.
![image](https://user-images.githubusercontent.com/54122306/194345436-47594ec7-842b-4555-bbc0-b2f40749f350.png)

**Comment by hoodriverheather** [comment]
@askarlupka good idea. We will look at the best way to track/indicate this in LabCAS.
@yuliujpl @nutjob4life @asitang We should discuss what is in Solr and the best way to display in the UI.
